### ISSUE TITLE
No binding by convention inside <Viewbox> with rc-155 +

### ISSUE THREAD

**Issue by borema** [Caliburn-Micro/Caliburn.Micro#741]
Binding by convention inside a `<Viewbox>` works as espected with rc-153 but starting with rc-155 it no longer works.

Here is some example code to illustrate the issue.

Viewmodel:

```
public class ShellViewModel : Conductor<object>
{
        public void TestBtn1()
        {
            MessageBox.Show("TEST 1");
        }

        public void TestBtn2()
        {
            MessageBox.Show("TEST 2");
        }
}

```

View:

```
<Grid>
        <Grid.RowDefinitions>
            <RowDefinition Height="0.5*" />
            <RowDefinition Height="0.5*" />
        </Grid.RowDefinitions>

        <Viewbox Grid.Row="0">
            <Button x:Name="TestBtn1" Content="Test1" />
        </Viewbox>

        <Button Grid.Row="1" x:Name="TestBtn2"  Content="Test2" />
</Grid>
```

**Comment by vb2ae** [comment]
I am unable to reproduce in a .net framework (4.7.2) WPF app with version 4.0.165-rc.   If you are still experiencing the issue could you please provide a sample that shows the issue.

**Comment by borema** [comment]
The issue seems to have been fixed with 4.0.165-rc from March 25.
